sequencer/src/state_signature.rs

```diff
@@ -79,3 +84,25 @@ impl StateSignatureMemStorage {
         self.pool.get(&height).cloned()
     }
 }
+
+/// Type for stake table commitment
+pub type StakeTableCommitmentType = (FieldType, FieldType, FieldType);
+
+/// Helper function for stake table commitment
+pub(crate) fn static_stake_table_commitment(
+    seed: [u8; 32],
+    num_nodes: usize,
+    capacity: usize,
+) -> (FieldType, FieldType, FieldType) {
+    let mut st = StakeTable::<BLSPubKey, StateVerKey, FieldType>::new(capacity);
+    (0..num_nodes).for_each(|i| {
+        let bls_key = BLSPubKey::generated_from_seed_indexed(seed, i as u64).0;
+        let state_key = StateKeyPair::generate_from_seed_indexed(seed, i as u64).ver_key();
```

Comment:
This might break when we change key management, so that nodes pass in their keys instead of generating them from seeds. This function should probably just take `known_nodes_with_stake` (ie the stake table itself) and compute the comitment from that

Reply:
Changed in 96d2af1